src/main/jbake/content/docs/components/btm/clientconfig.adoc

```diff
@@ -0,0 +1,81 @@
+= Client Configuration Guide
+Gary Brown
+2015-06-30
+:description: Hawkular Business Transaction Management
+:icons: font
+:jbake-type: page
+:jbake-status: published
+
+This guide will help you to configure a JVM based client environment to instrument a business application and send business transaction fragment (trace) data to the BTM server.
+
+== Configuring a Client Environment
+
+First step is to obtain the BTM distribution. If you have not already done so, then see the instructions on the link:serverinstall.html[Server Installation] page.
+
+Once the BTM distribution zip has been unpacked, you will find a folder called *_btm_*. This folder contains the necessary environment script, jar and instrumentation configuration files to configure a client JVM environment. 
+
+This section will explain how to use the supplied environment script, to setup the necessary client configuration, before starting the JVM. However if you would prefer to integrate this environment into your application/container configuration, then simply copy the entries from the contained *_env.sh_* script, moving the jar and configuration files into appropriate locations.
+
+First step is to setup the *_BTM_HOME_* environment variable to point to the *_btm_* folder in the unpacked BTM distribution.
+
+```
```

Comment:
If you use
[source,shell]
----
export ...
----

then AD is able to do syntax coloring. See e.g. https://github.com/hawkular/hawkular.github.io/blob/pages/src/main/jbake/content/docs/user/getting-started.adoc#install-a-wildfly-monitoring-agent